supervisor/supervisor.go

```diff
@@ -141,23 +142,25 @@ func (sv *Supervisor) getHyperPod(container string, spec *specs.Spec) (hp *Hyper
 		return nil, fmt.Errorf("The container %s is already existing", container)
 	}
 	for _, ns := range spec.Linux.Namespaces {
-		if ns.Path != "" {
-			if strings.Contains(ns.Path, "/") {
-				return nil, fmt.Errorf("Runv doesn't support path to namespace file, it supports containers name as shared namespaces only")
+		if len(ns.Path) > 0 {
+			pidexp := regexp.MustCompile(`/proc/(\d+)/ns/*`)
+			matches := pidexp.FindStringSubmatch(ns.Path)
+			if len(matches) != 2 {
+				return nil, fmt.Errorf("Can't find shared container with network ns path %s", ns.Path)
 			}
-			if ns.Type == "mount" {
-				// TODO support it!
-				return nil, fmt.Errorf("Runv doesn't support shared mount namespace currently")
-			}
```

Comment:
do you check the possible bugs when containers share mount namespace ?
if you fix it internal, please update it in this repo.
if you have enough confidence that it works well with share mount namespace(without this check), could you fire another pr to remove this check?